graph/loaders/comments.js

```diff
@@ -91,12 +91,19 @@ const getParentCountsByAssetID = (context, asset_ids) => {
 const getCommentCountByQuery = (context, {ids, statuses, asset_id, parent_id, author_id, tags, action_type}) => {
   let query = CommentModel.find();
 
-  if (ids) {
-    query = query.where({id: {$in: ids}});
+  if (
+    (context.user != null && context.user.can(SEARCH_NON_NULL_OR_ACCEPTED_COMMENTS)) ||
+    statuses && statuses.every((status) => ['NONE', 'ACCEPTED'].includes(status))
```

Comment:
If `statuses` is null/undefined, no comments will be returned. Is that desired?

Reply:
It'll return all comments if permissions are there, look inside the if clause

Reply:
For these if statements, if you can write a short comment above that describes how/why we're doing this. It'll make it super clear.

Reply:
👍